src/views/LogoutLedger.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="container">
+        <SmallPage>
+            <PageHeader>Mind the loss of settings</PageHeader>
+            <PageBody>
+                <div class="warning nq-text nq-red">
+                    Logging out will delete custom settings and labels. Contacts are not affected.
+                </div>
+                <div class="ledger-illustration"></div>
+                <div class="hint nq-text">Your Ledger is<br>required to log in again.</div>
+                <button class="logout-button nq-button red" @click="_logOut">Log Out</button>
+            </PageBody>
+
+            <Loader v-if="confirmedLogout"
+                state="success"
+                title="You are logged out."
+                class="grow-from-bottom-button">
+            </Loader>
+        </SmallPage>
+
+        <button class="global-close nq-button-s" @click="_close" :class="{ hidden: confirmedLogout }">
+            <span class="nq-icon arrow-left"></span>
+            Back to {{request.appName}}
+        </button>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { SmallPage, PageHeader, PageBody } from '@nimiq/vue-components';
+import { ParsedSimpleRequest } from '../lib/RequestTypes';
+import { WalletStore } from '@/lib/WalletStore';
+import { Static } from '../lib/StaticStore';
+import { ERROR_CANCELED } from '../lib/Constants';
+import Loader from '../components/Loader.vue';
+
+@Component({components: {SmallPage, PageHeader, PageBody, Loader}})
+export default class LogoutLedger extends Vue {
+    @Static private request!: ParsedSimpleRequest;
+
+    private confirmedLogout: boolean = false;
+
+    private async _logOut() {
```

Comment:
Methods in typescript really do not need to have underscores, and AFAIK no other component/view is using underscored private methods.

Reply:
We had a decision to mark private methods with underscores.